boards/nxp/frdm_mcxn947/frdm_mcxn947-pinctrl.dtsi

```diff
@@ -37,4 +37,26 @@
 			bias-pull-up;
 		};
 	};
+
+	pinmux_enet_qos: pinmux_enet_qos {
+		mdio_group {
+			pinmux = <ENET0_MDC_PIO1_20>,
+				<ENET0_MDIO_PIO1_21>;
+			slew-rate = "fast";
+			drive-strength = "low";
+			input-enable;
+		};
+		rx_group {
```

Comment:
nit- should name here should be something like `mac_group`? Since it has RX/TX pins